data/fixtures/settings.yml

```diff
@@ -12,7 +12,7 @@ QubitSetting:
     name: version
     editable: 0
     deleteable: 0
-    value: 192
+    value: 193
```

Comment:
Changing the data schema version doesn't seem necessary in this case, because the actual table schema isn't changing, just the data in the table. 🤔